pkgs/applications/networking/feedreaders/russ/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchCrate
+, rustPlatform
+, xorg
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "russ";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ckampfe";
+    repo = "russ";
+    rev = "1482bb1df13738fdd4ea1badf2146a9ed8e6656e";
+    hash = "sha256-MvTMo2q/cQ/LQNdUV8SmHgGlA42kLl0i9mdcoAFV/I4=";
+  };
+
+  cargoHash = "sha256-ObWrwXMGXkLqqM7VXhOXArshk2lVkbOTXhrQImDQp1s=";
+
+  nativeBuildInputs = [ xorg.libxcb ];
```

Comment:
This is a bit strange of a strange requirement for a non-graphical program, and it built fine for me without it.  What is it used for?

Reply:
I added it since the README for russ mentioned libxcb deps as a requirement.. it does seem to build fine without it so i'll get rid of this